### PR TITLE
Add config options for running behind proxies and with NAT

### PR DESCRIPTION
**Pull request by felinira**
This adds config options to allow for operation in a NAT'ed environment with a proxy that does SSL termination at the network border.

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -135,3 +135,10 @@ jitsi_meet_nginx_ssl_protocols: "{{ jitsi_meet_nginx_ssl_presets[jitsi_meet_ngin
 jitsi_meet_nginx_ssl_prefer_server_ciphers: "{{ jitsi_meet_nginx_ssl_presets[jitsi_meet_nginx_ssl_preset]['prefer_server_ciphers'] }}"
 jitsi_meet_nginx_ssl_ciphers: "{{ jitsi_meet_nginx_ssl_presets[jitsi_meet_nginx_ssl_preset]['ciphers'] }}"
 jitsi_meet_nginx_ssl_dhparam: "/etc/nginx/dhparam"
+
+# If you are behind a proxy that does SSL termination set this to false
+jitsi_meet_nginx_listen_https: true
```
Comment (xshadow) [inline defaults/main.yml:140]: Please change the variables to `jitsi_meet_nginx_listen_https_enabled`

File: defaults/main.yml [hunk 2]
```diff
@@ -135,3 +135,10 @@ jitsi_meet_nginx_ssl_protocols: "{{ jitsi_meet_nginx_ssl_presets[jitsi_meet_ngin
...
+
+# You need to set both jitsi_meet_nat_local_ip and jitsi_meet_nat_public_ip if you enable NAT
+jitsi_meet_behind_nat: false
```
Comment (xshadow) [inline defaults/main.yml:143]: Please change the variables to `jitsi_meet_behind_nat_enabled`

**Review by xshadow — CHANGES_REQUESTED** [review]
Thanks for your PR. Please change the variable names accordingly.

**Comment by felinira** [comment]
Done. Thanks for developing the role, it makes deploying jitsi really simple! I also configured the node exporter and grafana dashboards and they are really nice too :)

**Pull request merged**